features/steps/publisher.go

```diff
@@ -36,3 +62,58 @@ func (p *Publisher) logIn(username string) error {
 
 	return nil
 }
+
+func (p *Publisher) assignUserCookies() chromedp.Action {
```

Comment:
Might be my understanding but this is getting cookies right not assigning any cookies? The name might be a bit misleading.